File: test/fixtures/php-guzzle/post_xpost.php
<?php
require 'vendor/autoload.php';

use GuzzleHttp\Client;

$client = new Client();

$response = $client->post('http://localhost:28139/api/xxxxxxxxxxxxxxxx', [
    'headers' => [
        'Content-Type' => 'application/x-www-form-urlencoded'
    ],
    'body' => '{"keywords":"php","page":1,"searchMode":1}'
]);

File: test/fixtures/java-unirest/post_xpost.java
import kong.unirest.HttpResponse;
import kong.unirest.Unirest;

class Main {
    public static void main(String[] args) {
            HttpResponse<String> httpResponse
               = Unirest.post("http://localhost:28139/api/xxxxxxxxxxxxxxxx")
            .header("Content-Type", "application/x-www-form-urlencoded")
            .body("{\"keywords\":\"php\",\"page\":1,\"searchMode\":1}")
            .asString();
    }
}
